sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/util/Transforms.java

```diff
@@ -714,4 +736,33 @@ private static List<DocumentWord> toDocumentWords(
             })
             .collect(Collectors.toList());
     }
+
+    public static Map<String, com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails> toInnerDocTypes(Map<String, com.azure.ai.formrecognizer.documentanalysis.administration.models.ClassifierDocumentTypeDetails> tags) {
+        Map<String, com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails>
+            innerTags = new HashMap<String, com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails>();
+        tags.forEach((key, classifierDocumentTypeDetails) -> {
+            com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails innerClassifyDocTypeDetails
+                = new com.azure.ai.formrecognizer.documentanalysis.implementation.models.ClassifierDocumentTypeDetails();
+            if (classifierDocumentTypeDetails.getAzureBlobFileListSource() != null) {
+                innerClassifyDocTypeDetails.setAzureBlobFileListSource(new AzureBlobFileListSource().setContainerUrl(classifierDocumentTypeDetails.getAzureBlobFileListSource().getContainerUrl()).setFileList(classifierDocumentTypeDetails.getAzureBlobFileListSource()
+                    .getFileList()));
+            } else {
+                innerClassifyDocTypeDetails.setAzureBlobSource(new AzureBlobContentSource().setContainerUrl(classifierDocumentTypeDetails.getAzureBlobSource().getContainerUrl()).setPrefix(classifierDocumentTypeDetails.getAzureBlobSource().getPrefix()));
+            }
+            innerTags.put(key, innerClassifyDocTypeDetails);
+        });
+        return innerTags;
+    }
+
+    public static DocumentClassifierDetails toDocumentClassifierDetails(com.azure.ai.formrecognizer.documentanalysis.implementation.models.DocumentClassifierDetails inner) {
```

Comment:
NIT: 

might be thinking about using autorest customization. But I would leave it to you. low prioirty

Reply:
yep, we currently autorest does not allow placing the custom model (DocumentClassifierDetails ) in the `admin.models` subpackage.